AndroidManifest.xml

```diff
@@ -125,6 +125,12 @@
             android:exported="false"
             android:label="@string/search_users_and_groups_hint" />
 
+        <provider
+            android:name=".ui.adapter.DiskLruImageCacheFileProvider"
```

Comment:
That is a different functionality requiring specific tests. I will move it to a separate branch.

Please, include in the description of the PR a summary of all the changes. Otherwise the test plans could not match the scope of the changes, since we don't keep an strict order for the tasks of reviewing code and defining tests - tasks that should be independent.